Client.Linq/Internal/Expressions/String/Replace.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace InfluxDB.Client.Linq.Internal.Expressions.String
+{
+    internal class Replace : IExpressionPart
```

Comment:
Please rename to `ReplaceAll`
```suggestion
    internal class ReplaceAll : IExpressionPart
```